es-spatial-plugin/src/cmr/elasticsearch/plugins/spatial/factory/lfactory.clj

```diff
@@ -56,15 +57,14 @@
     (first (srl/ords-info->shapes ords-info ords))))
 
 (defn get-intersects-fn [script-params]
-  (try
-    (let [params (extract-params script-params)
-          shape (params->spatial-shape params)
-          intersects-fn (relations/shape->intersects-fn shape)]
-      (assert-required-parameters params)
-      intersects-fn)
-    (catch Throwable e
-      (.printStackTrace e)
-      (throw e))))
+  (let [params (extract-params script-params)
+        shape (params->spatial-shape params)]
+    (assert-required-parameters params)
+    (try
+      (relations/shape->intersects-fn shape)
+      (catch Exception e
+        (.error (LogManager/getLogger "cmr_spatial_lfactory") (format "Unable to create intersects function for shapes [%s]" (pr-str shape)) e)
```

Comment:
This is where the original exception was being thrown from. This adds extra logging information for better debugging.